.env.example

```diff
@@ -0,0 +1,6 @@
+GITHUB_TOKEN=
+WEB_PORT=8080
+LIBERAPAY_URL=https://liberapay.com/Tiled/public.json
+PATREON_URL=https://api.patreon.com/user/90066
+GITHUB_URL=https://api.github.com/graphql
```

Comment:
I'd suggest to make the accounts configurable instead of the URLs. So we need something like `LIBERAPAY_USERNAME`, `GITHUB_USERNAME` and `PATREON_ACCOUNT_ID` instead.